### PR TITLE
Feature/create logout view

### PR DESCRIPTION
**Pull request by BoroviyOrest**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=216)

## Code reviewers

- [x] @meelros
- [ ] @Kardan1123
- [ ] @mizin4ik
- [ ] @dimigor
- [ ] @PetrushynskyiOleksii
- [x] @Sasha1152

### Second Level Review

- [ ] @lhalam

## Summary of change

Add logout view and test for it, add cookies in login

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

File: way_to_home/custom_user/views.py [hunk 1]
```diff
@@ -97,7 +100,21 @@ def log_in(request):
     if not user:
         return RESPONSE_400_INVALID_EMAIL_OR_PASSWORD
     login(request, user=user)
-    return RESPONSE_200_OK
+    response = RESPONSE_200_OK
+
+    if data.get('save_cookies'):
+        response.set_cookie('user_id', user.id, max_age=TTL_USER_ID_COOKIE)
```
Comment (vo333ua) [inline way_to_home/custom_user/views.py]: for what do you need to save user id in cookie?

File: way_to_home/tests/unittests/custom_user/testviews.py [hunk 2]
```diff
@@ -178,32 +178,56 @@ def test_log_in(self):
...
+        """ Positive user logout test """
+        test_data = {
+            'email': 'user@mail.com',
+            'password': '1111Bb',
+        }
+        url_login = reverse('login_user')
+        response = self.client.post(url_login,
+                                    json.dumps(test_data),
+                                    content_type='application/json')
+        self.assertEqual(response.status_code, 200)
```
Comment (MaxDubchak) [inline way_to_home/tests/unittests/custom_user/testviews.py]: I'd advice you just use
`self.client.login(...)`

**Review by MaxDubchak — CHANGES_REQUESTED** [review]

**Review by dimigor — DISMISSED** [review]

**Review by Sasha1152 — DISMISSED** [review]

File: way_to_home/tests/unittests/custom_user/testviews.py [hunk 3]
```diff
@@ -178,32 +178,53 @@ def test_log_in(self):
...
         self.assertEqual(response.status_code, 400)
 
+    def test_logout(self):
+        """ Positive user logout test """
+        test_data = {
+            'email': 'user@mail.com',
+            'password': '1111Bb',
+        }
+        response = self.client.login(**test_data)
+        self.assertEqual(response, True)
```
Comment (MaxDubchak) [inline way_to_home/tests/unittests/custom_user/testviews.py]: remove assert, rename test_data  to login_credentials, don't store returned value of login method

**Review by MaxDubchak — DISMISSED** [review]
I approve

**Review by Sasha1152 — DISMISSED** [review]
Good!

**Review by meelros — APPROVED** [review]

**Review by Sasha1152 — APPROVED** [review]

**Review by dimigor — APPROVED** [review]

**Pull request merged**